documentation/asciidoc/topics/ref_upgrades.adoc

```diff
@@ -26,3 +26,25 @@ ifdef::downstream[]
 If you upgrade {brandname} clusters manually and have upgraded the channel for your {ispn_operator} subscription from **8.1.x** to **8.2.x** you should apply the upgrade for the latest {brandname} 8.2.x version as soon as possible to avoid potential data loss that can result from an issue in 8.2.0.
 For more information, see link:{relnotes_openshift}[{ispn_operator} {ProductVersion} Release Notes].
 endif::downstream[]
+
+[discrete]
+== Upgrade types
+//Community Only
+ifdef::community[]
+The Infinispan CR path ```spec.upgrades.type``` defines how the upgrades are done:
+
+[%header,cols=2*]
+|===
+|Value
+|Description
+
+|`HotRodRolling`
+|Upgrades the cluster with no downtime using Hot Rod Rolling Upgrades. The operator first creates a new statefulSet with pods in the new version, that use existing pods as a remote store; then redirects all clients to the new pods while copying all the data from the old cluster, disposing the original pods in the end.
+
+|`Shutdown`
+|This is the default migration type. The operator issues an orderly shutdown of the cluster, and pods are started with the new version.
```

Comment:
Upgrades {brandname} clusters with downtime. {ispn_operator} shuts down the existing cluster before removing it, then creates a new cluster with the target version. This is the default upgrade type.